### DISCUSSION TITLE
language&framework/java/2-java-data-type-and-variable-and-array

### DISCUSSION THREAD

**Discussion by giscus** [dev-ujin/dev-ujin.github.io#3] [Comments]
# language&framework/java/2-java-data-type-and-variable-and-array

⚓백기선 강사님의 live study에 나온 주제들을 혼자 공부하고 정리하여 작성한 글입니다.⚓

https://dev-ujin.github.io/language&framework/java/2-java-data-type-and-variable-and-array

**Comment by jki503** [comment]
For Server-side, memory should be especially strict! That's a good posting!

**Reply by devyoujin** [reply]
Thank you! 🌝
